styles/main.css

```diff
@@ -202,13 +204,13 @@ footer hr {
     grid-template-columns: 25% 25% 25% 25%;
 }
 
-section div a img {
+.gallery div a img {

```

Comment:
It's best to be more specific here.  The rule as you have it could effect sections other than the gallery section.